### PR TITLE
Change loss signature to match pytorch

### PR DESCRIPTION
**Pull request by mpariente**
#### Done
- Changed the loss functions and classes
- Changed `PITLossWrapper
- Changed the docs (signature and examples)
- Changed the tests [passing]
- Changed the PITLoss notebook
- Added a new test to be sure the permutation is the right one.

### CODE REVIEW

**Comment by popcornell** [comment]
No more issues with docstring then ?

**Comment by mpariente** [comment]
> No more issues with docstring then ?

Oh yes, just the variable names have been changed. We'll have to work on the docstrings but there'll be another branch for that
